components/home/__tests__/homepage-contributing.test.tsx

```diff
@@ -1,178 +1,10 @@
-            >
-              <svg
-                class="w-6 h-6 text-brand-500"
-                fill="none"
-                stroke="currentColor"
-                viewBox="0 0 24 24"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  d="M9 5H7a2 2 0 00-2 2v12a2 2 0 002 2h10a2 2 0 002-2V7a2 2 0 00-2-2h-2M9 5a2 2 0 002 2h2a2 2 0 002-2M9 5a2 2 0 012-2h2a2 2 0 012 2m-3 7h3m-3 4h3m-6-4h.01M9 16h.01"
-                  stroke-linecap="round"
-                  stroke-linejoin="round"
-                  stroke-width="2"
-                />
-              </svg>
-            </div>
-            <div
-              class="ml-4 space-y-1"
-            >
-              <h3
-                class="text-base leading-5 font-semibold"
-              >
-                <a
-                  class="helper-link-cover"
-                  href="https://www.indorelawan.org/activity/60e2ed45164da80018b0e246"
-                  rel="nofollow noopener noreferrer"
-                  target="_blank"
-                >
-                  Daftar Jadi Relawan
-                </a>
-              </h3>
-              <p
-                class="text-sm leading-4 text-gray-500"
-              >
-                Gabung jadi relawan untuk mengumpulkan data dan menjawab pertanyaan warga via hotline
-              </p>
-            </div>
-            <div
-              aria-hidden="true"
-              class="flex flex-none items-center justify-center ml-2"
-            >
-              <svg
-                class="w-6 h-6 text-brand-500"
-                fill="currentColor"
-                viewBox="0 0 20 20"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path
-                  clip-rule="evenodd"
-                  d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-                  fill-rule="evenodd"
-                />
-              </svg>
-            </div>
-          </li>
-        </ul>
-        <div
-          class="p-4 bg-blue-50 text-center rounded-md space-y-3"
-        >
-          <p
-            class="text-base leading-6 font-semibold"
-          >
-            Mau Upload Info?
-          </p>
-          <a
-            class="flex flex-row px-4 py-2 text-base rounded-full items-center justify-center border border-transparent font-medium shadow-sm focus:outline-none focus:ring-2 focus:ring-offset-2 text-brand-500 border-brand-500 hover:bg-blue-100 focus:ring-brand-500 disabled:cursor-not-allowed disabled:opacity-75"
-            href="https://docs.google.com/forms/d/e/1FAIpQLSc_m0WyokBk8xv9jBwYFN1zAt4_u-kE2TI9ZBrAokB4E08AGw/viewform"
-            rel="nofollow noopener noreferrer"
-            target="_blank"
-          >
-            Tambahkan info di sini
-          </a>
-        </div>
-      </section>
-    `);
+    expect(screen.getByText(/cara berkontribusi/gi)).toBeInTheDocument();
```

Comment:
Please assert more text along with the links that they're pointing at.

You can look at [my implementation here](https://github.com/kawalcovid19/wargabantuwarga.com/blob/85f7b0e0992cafcff406ad37ee30ff5b40c3872f/components/layout/__tests__/navigation.test.tsx#L153) as an example.